xbmc/pvr/windows/GUIWindowPVRTimers.cpp

```diff
@@ -223,6 +224,13 @@ bool CGUIWindowPVRTimers::OnMessage(CGUIMessage &message)
         Update(GetDirectoryPath());
         bReturn = true;
       }
+      else if (message.GetSenderId() == CONTROL_BTNHIDEDISABLEDTIMERS)
+      {
+        CSettings::GetInstance().ToggleBool(CSettings::SETTING_PVRTIMERS_HIDEDISABLEDTIMERS);
```

Comment:
Does this mean the setting can be toggled both from the sidebar and the system settings. IMO either is fine but not both. Do we have any other settings that behave like this?